### ISSUE TITLE
Default VirtIO-9p settings expose home directory of host user

### ISSUE THREAD

**Issue by piceae** [quickemu-project/quickemu#964]
Hi

#### Actual behaviour

quickemu "silently" uses default VirtIO-9p settings that expose the whole home of host user to the VM. In other words, the VM have access to whole home of the host user.

#### Expected behaviour

- documentation and warnings should be improved regarding the different sharing methods and the default options retained by quickemu.
- quickemu should restrict VirtIO-9p (and webdav) sharings to specific, empty by default, folders. Better, quickemu should not enable by default any host-to-guest or guest-to-host sharings. Or at least, it should allow users to deactivate any default sharing.

#### Steps to reproduce the behaviour

- start and login to a VM
- mkdir tmp
- sudo mount -t 9p -o trans=virtio,version=9p2000.L,msize=104857600 Public-<HOST_USERNAME> ~/tmp

#### Linux Distribution & Kernel

Arch Linux, 6.7.9.arch1-1

**Comment by lj3954** [comment]
I cannot replicate this issue. Only the ~/Public directory is shared with the guest, not the entire home. If the incorrect directory is being mounted, the issue is with your XDG directories. You can read more on this here: https://wiki.archlinux.org/title/XDG_user_directories. 

Certainly, though, a parameter should be added to disable all sharing, as well as a warning if the Public directory is equal to $HOME.

**Comment by piceae** [comment]
Hello,

Thanks for your time.

You are right, I deleted my Public folder long time ago since I did not use it. Then, a subsequent call to xdg-user-dirs-update (1) silently made my home folder the new public... What a strange behavior from this command !

On the other hand, quickemu is so amazing, fast and easy to use that some people are probably using it in production (so not only for quick and dirty tests). With this in mind:
- a better indication/warning of what is potentially shared by the host to the guest (because people like me generally will not read information about sharing methods if they do not plan to share something with the guest).
- I would also present all the options of quickemu in a standard way in the manpage (for now, some are in the standard "OPTIONS" section and some in an other non-standard subsection).
- Perhaps, sharing something by default is not the way to go, even if it is the "Public" folder?

Anyway, thanks a lot for your time and a big Thank you for all the contributors of this awesome project!
